app/views/home/small_bets.html.erb

```diff
@@ -0,0 +1,352 @@
+<header class="grid override border-b border-black grid-cols-1 bg-[#f0f0f0]">
+  <div class="relative flex items-center justify-center px-4 py-32 lg:px-[8vw] md:pb-20 md:pt-20">
+    <div class="flex flex-col max-w-screen-lg items-center gap-6 text-center z-10">
+      <%= image_tag "smallbets/smallbets.svg", alt: "Small bets", class: "h-48" %>
+      <h1 class="leading-none text-5xl md:text-7xl">You made your $1.<br>Now let's make many more.</h1>
+      <div class="text-xl max-w-md lg:text-2xl lg:max-w-3xl">Small Bets is a community of creators who support and learn from each other how to build a business, one small bet at a time.</div>
+      <div class="mt-2 flex flex-col sm:flex-row items-center justify-center gap-3 sm:gap-4 w-full max-w-[384px] sm:max-w-none sm:w-auto mx-auto">
+        <div class="w-full sm:w-auto">
+          <%= render "home/shared/button", text: "Join Small Bets", url: "https://dvassallo.gumroad.com/l/small-bets", variant: "light", size: "default" %>
+        </div>
+      </div>
+      <div class="text-lg text-muted mt-4">
+        Join <span class="font-medium text-black">6,947 amazing people</span> and get <br> immediate access to our support network.
+      </div>
+    </div>
+  </div>
+<div class="grid grid-cols-2 lg:grid-cols-4 w-full border-t border-black h-full override" role="list">
+  <div class="flex flex-col items-center justify-end p-4 md:p-6 h-full border-b border-black lg:border-b-0">
+    <%= image_tag "smallbets/hassan.png", alt: "Hassan", class: "w-32 h-32 object-contain mb-4" %>
+    <div class="text-base font-medium text-black text-center">Hassan runs 3 newsletters<br>with over 22k subscribers</div>
+  </div>
+    <div class="flex flex-col items-center justify-end p-4 md:p-6 h-full border-l lg:border-x border-black border-b lg:border-b-0">
+    <%= image_tag "smallbets/greg.png", alt: "Greg", class: "w-32 h-32 object-contain mb-4" %>
+    <div class="text-base font-medium text-black text-center">Greg made $50k<br>writing tech books</div>
+  </div>
+  <div class="flex flex-col items-center justify-end p-4 md:p-6 h-full">
+    <%= image_tag "smallbets/steph.png", alt: "Steph", class: "w-32 h-32 object-contain mb-4" %>
+    <div class="text-base font-medium text-black text-center">Steph sells Cinema4D<br>material packs</div>
+  </div>
+  <div class="flex flex-col items-center justify-end border-l border-black p-4 md:p-6 h-full">
+    <%= image_tag "smallbets/xinran.png", alt: "Xinran", class: "w-32 h-32 object-contain mb-4" %>
+    <div class="text-base font-medium text-black text-center">5K people just signed to<br>Xinran's AI course</div>
+  </div>
+</div>
```

Comment:
Let's skip this section for now. I really like the idea, but I want to gather some compelling and diverse statements and I don't want it to block the whole thing.